meeting-notes/2021-02-15-planning.md

```diff
@@ -0,0 +1,95 @@
+# The Spark Planning 2021-02-15
+
+@alanshaw
+- 🚢 Shipped:
+    - [Team operations document](https://github.com/protocol/the-spark/pull/1)
+- ⏭ Next:
+    - Write up project proposals
+    - Wednesday is project prioritization day
+        - Review existing projects & pick top 5
+
+@hugomrdias 
+- 🚢 Shipped:
+    - Shipping js-ipfs stuff to focus on this team work
+- ⏭ Next:
+    - Working on pitches
+
+@gozala
+- 🚢 Shipped:
+- ⛔️ Blocked:
+- ⏭ Next:
+    - Submit pitches (listed below)
+    - Help with process definition for pitch submissions
+    - Help wrap up TS work
+
+@mgoelzer
+- 🚢 Shipped:
+- ⛔️ Blocked:
+- ⏭ Next:
+    - Submitted one project pitch to [the doc](https://docs.google.com/document/d/1lgIMxA-OyTe5IIQuhXvPP7GjL6DurbLZesBOm4s-aJE/edit)
+    - Still figuring out my responsibilities with this team, PM team, etc
+
+@riba
+- ⏭ Next:
+    - Still doing a number of wrap-ups on the sqlotus + unblocking the filplus+verifdeal+sentinel-backfill flow
+    - Attempt to write a proposal spin-off sqlotus, still unclear whether worthwhile
+    - Sync with team-members on their proposals
+
+## Projects
+
+Stages key:
+
+1. 💡 Researching/pitching
+2. ⏳ Evaluating (Pitch PR sent)
+3. 🚜 In Progress (Working on project)
+4. 🤝 Handover/feedback (Submitted back to customer)
+5. ✨ Complete
+
+### Personal retrieval miner aaS (💡) (@alanshaw)
+
+@alanshaw
+- Flesh out the pitch
+
+### E2E tests for deal flow in lotus (💡) (@alanshaw)
+
+@alanshaw
+- Flesh out the pitch
+
+### CID bounty - make the miners come to you (💡) (@alanshaw)
+
+@alanshaw
+- Flesh out the pitch
+
+@mgoelzer
+- (+1 for inspiration value - @mike)
+
+### [Symbiotic Networks](https://hackmd.io/@gozala/symbiotic-network) (💡) (@gozala)
+
+@gozala 
+- Propose session for presentation of the proposal
+- Complete the proposal
+
+### [Reliable Mutability Primitive](https://hackmd.io/@gozala/ipns-pitch) (💡) (@gozala)
+
+@gozala 
+- Complete the proposal
+
+### [Unified FileSystem API](https://hackmd.io/@gozala/unified-fs-api) (💡) (@gozala)
+
+@gozala 
+- Complete the proposal
+
+@alanshaw 
+- Long time ago I made a proposal: https://github.com/ipfs-inactive/interface-js-ipfs-core/pull/552
+    - Like the idea of `/mfs` - it was mentioned here also!
+
+### [Infrastructure coop](https://hackmd.io/@gozala/infrastructure-coop) (💡) (@gozala)
+
+@gozala 
+- Complete the proposal
+
+### [Remote wallet support](https://docs.google.com/document/d/1lgIMxA-OyTe5IIQuhXvPP7GjL6DurbLZesBOm4s-aJE/edit#heading=h.k18qpvwv6xrh) (@mgoelzer)
+
+@mgoelzer
+ - Next step: Ready for prioritization meeting
+ - Send PR to https://github.com/protocol/web3-dev-team
```

Comment:
@mgoelzer I added this but I believe you were going to sync with Alex to see if he was planning to PR the proposals from the doc.